### DISCUSSION TITLE
Passing Cypress results to Jira ticket by ticket ID

### DISCUSSION THREAD

**Discussion by radovanBabjak** [cypress-io/cypress#19742] [CI setup]
Is it possible to pass Jira ticket ID through GitHub-actions to Cypress in order to **show the result of the test on a given ticket**?

I am using manual workflow and have a subscription to the Cypress dashboard.

Thanks.

name: GitHub Actions Cypress Test
on:
 workflow_dispatch:
  inputs:
   JiraID:
    description: 'Jira ticket ID'
    default: 'SX-XXXX'
    required: true

**Comment by penchef** [comment]
Hey @radovanBabjak!

That shouldn't be too hard with the jira action https://github.com/atlassian/gajira . You can make the comment conditional like

```
.... // run cypress test
- name: Comment on Jira issue (failure)
  if: failure() 
  ... // comment: "Failure :("
- name: Comment on Jira issue (success)
  if: success() 
  ... // comment: "Success :("
